modules/katello_repository.py

```diff
@@ -212,9 +168,13 @@ def main():
 
     ping_server(module)
 
+    entity_dict['organization'] = find_organization(module, name=entity_dict['organization'])
+
+    entity_dict['product'] = find_product(module, name=entity_dict['name'], organization=entity_dict['organization'], failsafe=True)
```

Comment:
You might want to fail here if the product cannot be found (without failsafe).

Reply:
that looks to have been it. However....

```TASK [Ensure product 'Test Product' is present] ********************************
ok: [fixtures]
``` and if i go look in the UI its there.
![test_product](https://user-images.githubusercontent.com/372939/34725709-5e83c652-f517-11e7-8dc6-6cc0ad241a17.png)

Reply:
Trying to make the pdb bit working